ssv/committee_runner.go

```diff
@@ -208,27 +229,87 @@ func (cr CommitteeRunner) ProcessPostConsensus(signedMsg *types.PartialSignature
 			copy(specSig[:], sig)
 
 			if role == types.BNRoleAttester {
+				// Get the beacon object related to root
 				att := beaconObjects[root].(*phase0.Attestation)
+
+				// Insert signature
 				att.Signature = specSig
-				// broadcast
+
 				if err := cr.beacon.SubmitAttestation(att); err != nil {
 					return errors.Wrap(err, "could not submit to Beacon chain reconstructed attestation")
 				}
+				cr.RecordSubmission(types.BNRoleAttester, validator, slot)
```

Comment:
This can be moved after if-else, by using `role` as the first parameter...

Note to self: need to check if nothing is missing here besides `att.Signature`

Reply:
Yes! Thanks!

Reply:
I think maybe in the sync committee `syncMsg.ValidatorIndex = validator` is extraneous... you have a good test for this correct?

Reply:
so delete this line and make sure tests pass correctly